data_processors/pipeline/lambdas/umccrise.py

```diff
@@ -97,7 +85,7 @@ def handler(event, context) -> dict:
     logger.info(f"Start processing {WorkflowType.UMCCRISE.name} event")
     logger.info(libjson.dumps(event))
 
-    subject_id = event['subject_identifier_umccrise']
+    subject_id = event['subject_identifier']
     sample_name = event['sample_name']  # TUMOR_SAMPLE_ID
```

Comment:
We also need the sample ids to go in as inputs. For the umccrise tool these are dragen_tumor_id and dragen_normal_id, these should match the sample name used (RGSM in fastq list rows). UMCCRise requires these match the values in the VCF

Reply:
Sure; got it. I will update it appropriate rgsm/sample_id mapping there, yes.

Reply:
Looks like `wfr.3bb2f749f96f44eaa6e3c67ec43d6111` succeeded. I will push my changes in tick, next. Let us review each other, ok.

Reply:
@alexiswl Some improved made with https://github.com/umccr/data-portal-apis/commit/41b70ff28a97f4b7c64b6e309d1784e7a6e9aa4f Please kindly check it at your end.

Reply:
@williamputraintan T/N payload has changed with this PR. So T/N `/pairing` endpoint response will be changed as well. You can peek the diff here. Something to follow up on UI T/N trigger. After this get merged.

Reply:
@reisingerf Added you as second pair of eye for this, pls.

Reply:
Amazing, thanks @victorskl